mmv1/templates/terraform/constants/workbench_instance.go.tmpl

```diff
@@ -243,23 +243,23 @@ func resizeWorkbenchInstanceDisk(config *transport_tpg.Config, d *schema.Resourc
 	return nil
 }
 
-// mergeLabels takes two maps of labels and returns a new map with the labels merged.
-// If a key exists in old_labels but not in new_labels, it is added to the new map with an empty value.
-func mergeLabels(oldLabels, newLabels map[string]interface{}) map[string]string {
-	modifiedLabels := make(map[string]string)
+// mergeMaps takes two maps and returns a new map with the merged results.
+// If a key exists in old_maps but not in new_maps, it is added to the new map with an empty value.
+func mergeMaps(oldMaps, newMaps map[string]interface{}) map[string]string {
```

Comment:
Could the variable names be singular since each one is a single map?

```suggestion
func mergeMaps(oldMap, newMap map[string]interface{}) map[string]string {
```